### ISSUE TITLE
Accessibility of manager: Keyboard focus indication

### ISSUE THREAD

**Issue by TimGS** [ClipperCMS/ClipperCMS#197]
This arises from #194.

Can we have some indication of keyboard focus on links (including tabs) ?

Chrome was doing this (via an outline), though not in an aesthetically ideal manner, hence it's removal in #194.

**Comment by TimGS** [comment]
There's probably no rush for this - a fix for #198 is also required for any CSS changes for this issue to be effective.

**Comment by microcipcip** [comment]
Do you mean focus of input fields or just links/tabs? For input fields is already done on line 444-455 of style.css, although is not much visibile:

```
form#mutate input.inputBox:focus,
form#mutate input:focus,
form#mutate textarea:focus,
div.tmplvars input:focus,
form#mutate input.inputBox:hover,
form#mutate textarea:hover,
div.tmplvars input:hover{
    border-color: #C0C0C0 #D6D6D6 #D6D6D6 #C0C0C0;
    -moz-box-shadow: 0 0 4px #DDD;
    -webkit-box-shadow: 0 0 4px #DDD;
    box-shadow: 0 0 4px #DDD;
}
```

For the link there is already line 135-137 of style.css:

```
a:hover {
    color: #0f1e76;
}
```

**Comment by TimGS** [comment]
Just keyboard focus for links/tabs - though maybe the input focus should be stronger (?).

Can we change 'a:hover' to 'a:hover, a:focus' ?

**Comment by microcipcip** [comment]
I guess so, I will try and see. What would be the usage for a:focus? When you open Tvs and Chunks?

**Comment by TimGS** [comment]
For keyboard-only users, any link - most notably tabs and the top menu.

Due to #198, this is academic at the moment, but both issues need fixing at some point.
